public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx

```diff
@@ -98,8 +116,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       },
     }),
     selectedBox: css({
-      border: `2px solid ${theme.colors.primary.main}`,
-      boxShadow: `0 0 0 1px ${theme.colors.primary.main}`,
+      boxShadow: `inset 0 0 2px 2px ${theme.colors.primary.main}`,
```

Comment:
think we should remove this border, we usually never user blue border for selected state only focus state (And that already exists for this card) (and it's the wrong color, primary.main is a background token not a border token) , w